### PR TITLE
[FrameworkBundle][Scheduler] Add test for autoconfigured schedule

### PR DESCRIPTION
**Pull request by valtzu**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| License       | MIT

Add functional tests for autoconfigured schedules / tasks, as discussed in https://github.com/symfony/symfony/pull/51525#discussion_r1342467103.

### CODE REVIEW

**Comment by valtzu** [comment]
Added autoconfigured scheduler tests as discussed in https://github.com/symfony/symfony/pull/51525#discussion_r1342467103, but those are failing in the pipeline due to missing cron dev-dependency. Should I just remove cron attributes from DummyTask or how to proceed here @fabpot ?

Locally with `dragonmantank/cron-expression` installed the tests pass just fine.

**Comment by fabpot** [comment]
> Added autoconfigured scheduler tests as discussed in [#51525 (comment)](https://github.com/symfony/symfony/pull/51525#discussion_r1342467103), but those are failing in the pipeline due to missing cron dev-dependency. Should I just remove cron attributes from DummyTask or how to proceed here @fabpot ?
> 
> Locally with `dragonmantank/cron-expression` installed the tests pass just fine.

You can add `dragonmantank/cron-expression` as a `dev` dependency for the bundle and in the main `composer.json` file.

File: src/Symfony/Bundle/FrameworkBundle/composer.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
         "symfony/uid": "^5.4|^6.0|^7.0",
         "symfony/web-link": "^5.4|^6.0|^7.0",
         "phpdocumentor/reflection-docblock": "^3.0|^4.0|^5.0",
+        "dragonmantank/cron-expression": "^3.1",
```
Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/composer.json]: You need to move it to keep the list ordered.
Also, you must add it to the main `composer.json` file as well.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/FrameworkBundle/composer.json (hunk 1 above)
Comment (valtzu) [reply src/Symfony/Bundle/FrameworkBundle/composer.json]: I suspected that but then saw `phpdocumentor` after `symfony` :thinking: 

Fixed the order, and it is already added to main composer.json previously

**Review by OskarStark — APPROVED** [review]
Tests are 💚 so I think this can be merged @nicolas-grekas @fabpot

**Comment by fabpot** [comment]
Thank you @valtzu.

**Pull request merged**